Formula/aften.rb

```diff
@@ -13,10 +13,22 @@ class Aften < Formula
 
   depends_on "cmake" => :build
 
+  resource "sample_wav" do
+    url "http://www.mediacollege.com/audio/tone/files/1kHz_44100Hz_16bit_05sec.wav"
+    sha256 "949dd8ef74db1793ac6174b8d38b1c8e4c4e10fb3ffe7a15b4941fa0f1fbdc20"
+  end
+
   def install
     mkdir "default" do
       system "cmake", "-DSHARED=ON", "..", *std_cmake_args
       system "make", "install"
     end
   end
+
+  test do
+    assert_match %r{Aften: A\/52 audio encoder}, shell_output("#{bin}/aften -h").lstrip.lines.map(&:chomp).first
+
+    resource("sample_wav").stage testpath
```

Comment:
We already have a `.wav` test object available. Can we use that one or is it problematic in this case? The usage would be something like:

```ruby
system "#{bin}/aften", test_fixtures("test.wav"), "sample.ac3"
```